policy/release/run.sh

```diff
@@ -0,0 +1,5 @@
+for i in `ls *.rego | grep -v test`; do
```

Comment:
Unintentional addition?

Reply:
yes, lol. I used that to create the directories. removed.